ground/src/main/java/com/google/android/ground/repository/OfflineAreaRepository.kt

```diff
@@ -218,7 +218,10 @@ constructor(
     val baseUrl = getFirstTileSourceUrl()
     val mogCollection =
       MogCollection(
-        listOf(MogSource("${baseUrl}/world.tif", 0..7), MogSource("${baseUrl}/{x}/{y}.tif", 8..14))
+        listOf(
+          MogSource("${baseUrl}/world-masked.tif", 0..7),
+          MogSource("${baseUrl}/{x}/{y}.tif", 8..14)
+        )
```

Comment:
Should we consider creating a cons field for these params and moving to the central Config file?

Reply:
I'd much rather move them directly into the survey offline map sources, but I can move to a single constant for now to make the deps clearer.